### ISSUE TITLE
Repetitive product usage description in UG

### ISSUE THREAD

**Issue by matthewken19** [matthewken19/pe#1]
Good job on the writing of the UG! However, the repetitive product usage description can be difficult to follow and possibly confusing to the user. 

![image.png](https://raw.githubusercontent.com/matthewken19/pe/main/files/2b38c6a2-e5c9-4572-b68a-6e265e1e5fae.png)

The description "adding, deleting, editing, viewing, and finding of contacts, as well as adding, deleting, editing, viewing, and listing ongoing events" becomes confusing as the difference between managing contacts and events (finding contacts vs listing ongoing events) is not clear as the repetition of "adding, deleting, editing, viewing" makes it hard to follow and spot the difference.

**Comment by nus-se-script** [comment]
# Team's Response
We reject this bug. The reasons are as follows:

* A close reading of the sentence would not yield such ambiguity. 

* The sentence is meant to be complete in terms of describing the application.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [x] I disagree

**Reason for disagreement:** Based on the course website, the UG description above has unnecessary repetitions and users might easily miss the difference between `finding` contacts and `listing` ongoing events. A close reading would indeed make it clearer, but most users would not spend too much time reading the UG and if the users need to pay such attention in the details of the UG, it means that the UG description is not clear enough.

![image.png](https://raw.githubusercontent.com/matthewken19/pe/main/files/b25da06a-798b-42ad-9a54-6d956f68057a.png)


<catcher-end-of-segment><hr>
